src/components/scores/table.cjsx

```diff
@@ -154,6 +154,11 @@ module.exports = React.createClass
       if heading.type is 'homework'
         average =
           <span className="average">---</span>
+      else if heading.type is 'external'
+        average =
+          <span className="click-rate">
+            {(heading.completion_rate * 100).toFixed(0)}% have clicked link
```

Comment:
Does this need the same `0%`-means-no-students-responded logic that was added elsewhere recently?